src/construct.yaml

```diff
@@ -0,0 +1,25 @@
+{% from "version.jinja" import version %}
+{% from "python.jinja" import py_ver_maj, py_ver_min %}
+
+
+name: miniforge-py{{ py_ver_maj }}{{ py_ver_min }}
+version: {{ version }}
+
+install_in_dependency_order: true
+
+channels:
+  - http://conda.anaconda.org/conda-forge
+  - https://repo.continuum.io/pkgs/main
+  - https://repo.continuum.io/pkgs/free
+  - https://repo.continuum.io/pkgs/r
+  - https://repo.continuum.io/pkgs/pro
+  - https://repo.continuum.io/pkgs/msys2
```

Comment:
These have the same ordering that `conda` uses. For Linux and macOS, `conda-forge` is sufficient for making the installer. For Windows, we need the other channels in order to include things like `vs2008_runtime` for Python 2. Though for Python 3 on Windows, we already have the needed `vs2015_runtime` in `conda-forge`.